### DISCUSSION TITLE
Page build and commit successful, shows correctly in repo, but not seeing changes on site

### DISCUSSION THREAD

**Discussion by dlautman** [community/community#21928] [Other Feature Feedback, Questions, & Ideas]
<p>I recently made a change to my site’s footer. When I look at the files in the “_site” folder, the show up correctly. However, when I look at the actual page on <a href="http://github.io" rel="nofollow noopener">github.io</a>, it does not show the updated footer.</p>
<p>Example:</p>
<p><a href="https://github.com/dlautman/dlautman.github.io/blob/master/_site/index.html" rel="nofollow noopener">https://github.com/dlautman/dlautman.github.io/blob/master/_site/index.html</a></p>
<p>note lines 207 and 209 have my social media links.</p>
<p>Compare to page on <a href="http://github.io" rel="nofollow noopener">github.io</a>:</p>
<p><a href="http://dan.lautman.net/index.html" rel="nofollow noopener">http://dan.lautman.net/index.html</a></p>
<p>If you look at the source you will see the links are not there.  There are also some other discrepancies (like jekyll seo tags missing). What’s weird though is that other changes do show (for example I updated the post.html layout and the change went through no problem). </p>
<p>What am I missing here? </p>
<p>edit: in researching, I see notes about making sure this is in the master branch. I have confirmed it is.</p>
<p>Another thing: I recently updated to the latest version of minima (not through the Gemfile). However I don’t think that is related because clearly the site is building correctly and the source code appears correct on the repo, just not on <a href="http://github.io" rel="nofollow noopener">github.io</a>.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Taking a look at the repo and the website, it appears that everything is working now? It looks like you stopped using the _site directory and used the theme to get the links you wanted?</p>

**Comment by amitai** [comment]
<p>sorry: posted from wrong account!</p>
